File: Programming Language(ver.2)/C++/204.cpp
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    char sign;
    cin >> a >> b >> sign;
    switch (sign)
    {
    case '+':
        cout << a << "+" << b << "=" << (a + b);
        break;
    case '-':
        cout << a << "-" << b << "=" << (a - b);
        break;
    case '*':
        cout << a << "*" << b << "=" << (a * b);
        break;
    default:
        cout << "error";
        break;
    }
    return (0);
}

File: Programming Language(ver.2)/Python/204.py
a = int(input())
b = int(input())
compute = input()

if compute == '+':
    print(str(a) + '+' + str(b) + '=' + str(a+b))
elif compute == '-':
    print(str(a) + '-' + str(b) + '=' + str(a-b))
elif compute == '*':
    print(str(a) + '*' + str(b) + '=' + str(a*b))
else:
    print('error')